pages/api/admin/users/index.js

```diff
@@ -0,0 +1,56 @@
+import { handleRequests } from "@klaudsol/commons/lib/API";
+import { withSession } from "@klaudsol/commons/lib/Session";
+import { createHash } from '@/lib/Hash';
+import { OK, NOT_FOUND } from '@klaudsol/commons/lib/HttpStatuses';
+import InsufficientDataError from '@klaudsol/commons/errors/InsufficientDataError';
+import UserAlreadyExists from "@klaudsol/commons/errors/UserAlreadyExists";
+import People from '@klaudsol/commons/models/People';
+import PeopleGroups from '@klaudsol/commons/models/PeopleGroups';
+
+export default withSession(handleRequests({ get, post }));
+
+async function get(req, res) {
+    const { approved, pending } = req.query;
```

Comment:
Kindly use assertUserCan at the beginning of all API endpoints.

https://github.com/klaudsol/klaudsol-commons/blob/main/src/lib/Permissions.js#L78